target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -124,6 +124,12 @@ tplink,tl-wr902ac-v3)
 	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x10"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x10"
 	;;
+tuoshi,lt22m)
+	ucidef_set_led_switch "lan0" "lan0" "blue:lan0" "switch0" "0x1"
+	ucidef_set_led_switch "lan1" "lan1" "blue:lan1" "switch0" "0x2"
+	ucidef_set_led_netdev "wan" "wan" "blue:4g" "eth1"
```

Comment:
`eth1` here is the interface created by USB CDC RNDIS driver as that's what the LTE module offers by default.

Reply:
@paulfertser which means the link is always up. Any idea how to make it detect WWAN link status?

Reply:
The link is "up" as soon ad the modem is powered and the kernel has detcted the device. The modem seems to support "rndis" mode only. "rndis_host" does not implement link events (it doubt it's even possible because the information seems only to be available via AT commands).

I'd entirely remove the default mapping in the first commit and investigate later in how to support the modem better. My idea is to improve the support of modem status (Link detection, IP change detection, mode detection, LED support) in https://github.com/openwrt/openwrt/tree/master/package/network/utils/comgt in a separate PR.